packages/react-dom/src/__tests__/ReactTestUtilsAct-test.js

```diff
@@ -779,13 +779,5 @@ function runActTests(render, unmount, rerender) {
         });
       }
     });
-    describe('throw in prod mode', () => {
```

Comment:
Replaced by the test in `packages/react-reconciler/src/__tests__/ReactIsomorphicAct-test.js`. This was always testing `React.act` not `react-dom/test-utils`